src/components/HeaderName.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+type Header = {
+  name: "string";
+};
+
+export function HeaderTitle(props: Header) {
```

Comment:
Component name doesn't match file name, also what about the account icon that is also on the header in the mockup?
![image](https://user-images.githubusercontent.com/43496778/222137103-32a4aefe-04a6-4002-9ac9-2022c2b2471d.png)

Reply:
ignore the icon comment, I just saw it's part of another issue & PR,
please address the differing file/component names